internal/router/router.go

```diff
@@ -18,6 +19,14 @@ func CreateAPIRouter(api api.ProxyAPI) *httprouter.Router {
 	return proxyRouter
 }
 
+// CreateJenkinsAPIRouter is creating a router for the REST API of the Proxy.
+func CreateJenkinsAPIRouter(jenkinsAPI jenkinsapi.JenkinsAPI) *httprouter.Router {
+	// Create router for API
+	jenkinsAPIRouter := httprouter.New()
+	jenkinsAPIRouter.GET("/api/jenkins/start", jenkinsAPI.Start)
```

Comment:
this should be `POST` instead of a `get` since we are changing the state of a resource.